app/src/main/AndroidManifest.xml

```diff
@@ -44,7 +44,6 @@
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.mydownloads.MyDownloadsActivity"
-      android:screenOrientation="portrait"
```

Comment:
Suggest introducing a label for accessibility purposes.

`<string name="my_downloads_activity_title">My Downloads</string>`

Reply:
Also add test for the same.

Reply:
done